sklearn/lda.py

```diff
@@ -1,6 +1,20 @@
 import warnings
+from .discriminant_analysis import LinearDiscriminantAnalysis as _LDA
```

Comment:
Previously it exposed `LinearDiscriminantAnalysis` in the public API under `sklearn.lda`. I'm not sure we want that. I remember us having some problems having done something similar for warning classes...